packages/mobile/src/onboarding/registration/PictureInput.tsx

```diff
@@ -0,0 +1,121 @@
+import colors from '@celo/react-components/styles/colors'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Image, Platform, StyleSheet, Text, View } from 'react-native'
+import { TouchableOpacity } from 'react-native-gesture-handler'
+import ImagePicker from 'react-native-image-crop-picker'
+import OptionsChooser from 'src/components/OptionsChooser'
+import { Namespaces } from 'src/i18n'
+import Photo from 'src/icons/Photo'
+import Logger from 'src/utils/Logger'
+
+interface Props {
+  picture: string | null
+  onPhotoChosen: (dataUrl: string | null) => void
+  backgroundColor: string
+}
+
+function PictureInput({ picture, onPhotoChosen, backgroundColor }: Props) {
+  const [showOptions, setShowOptions] = useState(false)
+  const updateShowOptions = (show: boolean) => () => setShowOptions(show)
+
+  const { t } = useTranslation(Namespaces.accountScreen10)
+
+  const pickPhoto = async (pickerFunction: typeof ImagePicker.openPicker) => {
+    try {
+      const image = await pickerFunction({
+        width: 200,
+        height: 200,
+        cropping: true,
+        includeBase64: true,
```

Comment:
We do get the path, but we need the data to store it ourselves, otherwise if the user deletes or moves the picture from their phone it will stop working.

Reply:
Ok yes, what I was suggesting is we only take the path and make a copy using `RNFS` and avoids transferring the actual content over the RN bridge.
Anyway let's not worry about this for now given the limited image size.